ckanext/dataset_subscriptions/templates/email_body.j2

```diff
@@ -0,0 +1,15 @@
+{% set num = activities|length %}
+{{ _("Here's a list of recently updated datasets you're following in DMS:") }}
+
+{% set counter = 1 -%}
```

Comment:
this looks like an offset not a counter. Is that right?

Reply:
even better use `loop.index` as https://jinja.palletsprojects.com/en/3.0.x/templates/